### PR TITLE
Add article-list and article-cards components

### PR DESCRIPTION
**Pull request by zrebcu411**

### CODE REVIEW

File: components/desktop/KnitArticleList.vue [hunk 1]
```diff
@@ -1,17 +1,79 @@
...
 
 export default {
+  data () {
+    return {
+      articles: [
+        {
+          title: 'Rozpoczęła się rejestracja na Mocarne Warsztaty!',
+          author: 'Grzegorz Nowakowski',
+          content: 'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laboriosam perspiciatis voluptatibus beatae quam, doloribus minus!',
+          articleThumbnailUrl: '/temporary/card-photo-0.jpg',
```
Comment (k911) [inline components/desktop/KnitArticleList.vue]: I think that writing prefix like article in article object is really redundant

File: components/desktop/KnitArticleList.vue [hunk 2]
```diff
@@ -1,17 +1,79 @@
...
+          authorAvatarUrl: '/temporary/article-author.png',
+          publishedAt: '2017-10-25T15:19:21+00:00',
+          numberOfLikes: 18,
```
Comment (k911) [inline components/desktop/KnitArticleList.vue]: The same as above: likes is so much simpler

**Comment by k911** [comment]
I left two small suggestions. :)

File: components/desktop/KnitArticleList.vue (hunk 1 above)
Comment (FieryCod) [reply components/desktop/KnitArticleList.vue]: It is unnecessary for now

**Comment by FieryCod** [comment]
I'm not convinced about validating props. I think we should delete validation.

File: components/desktop/KnitArticleCard.vue [hunk 3]
```diff
@@ -1,18 +1,225 @@
...
+// @TODO: Make elipsis overflow in article card contend
+
+export default {
+  data () {
+    return {
+      isLiked: false
+    }
+  },
+  props: {
+    title: {
```
Comment (FieryCod) [inline components/desktop/KnitArticleCard.vue]: Please delete this validation.

**Pull request merged**